aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -419,6 +425,20 @@ Vec256<int16_t> inline operator-(const Vec256<int16_t>& a, const Vec256<int16_t>
   return _mm256_sub_epi16(a, b);
 }
 
+// Negation. Defined here so we can utilize operator-
+
+Vec256<int64_t> Vec256<int64_t>::neg() const {
```

Comment:
There does exist an [xor instruction](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#techs=AVX,AVX2&text=xor&expand=5189,6125) for integers as well (for AVX2+). - This could aid further optimization.